dss-example/pom.xml

```
@@ -12,6 +12,14 @@
     <artifactId>dss-example</artifactId>
     <packaging>jar</packaging>
     <version>0.4.4</version>
+    <dependencies>
+        <dependency>
+            <groupId>io.github.ztkmkoo</groupId>
+            <artifactId>dss-server</artifactId>
+            <version>0.4.4</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

```suggestion
<artifactId>dss-example</artifactId>
    <packaging>jar</packaging>
    <version>0.4.4</version>

    <name>DSS EXAMPLE</name>
    <description>Distributed server system</description>
    <url>https://github.com/ztkmkoo/dss</url>
    
    <dependencies>
    	<dependency>
    		<groupId>${project.groupId}</groupId>
    		<artifactId>dss-server</artifactId>
    		<version>0.4.4</version>
    		<type>jar</type>
           	<scope>compile</scope>
		</dependency>
    </dependencies>
```

이미 이전 dss-example/pom.xml에서 dss-server에 대한 dependency가 있어 굳이 바꿀 이유가 없어보입니다.